.github/workflows/gitleaks.yaml

```diff
@@ -0,0 +1,19 @@
+name: gitleaks
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *" # run once a day at 4 AM
```

Comment:
(blocking) GitHub actions run in UTC. Can you update the time to be 4 AM EST/EDT and update the comment to include the time it runs in EST/EDT and PST/PDT. It make it less confusing that having to try to remember what it should be running at each time you open the file.

Reply:
Done

Reply:
Done